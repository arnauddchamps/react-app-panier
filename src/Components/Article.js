// TODO Article qui affiche une ligne du panier

import React from 'react'

const Article = () => <div>
  <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Zinedine_Zidane_by_Tasnim_03.jpg/280px-Zinedine_Zidane_by_Tasnim_03.jpg" alt="zidane"/>
</div>


export default Article