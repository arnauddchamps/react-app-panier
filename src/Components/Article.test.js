import React from 'react';
import ReactDOM from 'react-dom';
import Article from './Article';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Article />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('renders correctly', () => {
  const tree = renderer
    .create(<Article />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});