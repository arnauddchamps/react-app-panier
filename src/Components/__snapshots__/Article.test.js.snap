// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt="zidane"
    src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Zinedine_Zidane_by_Tasnim_03.jpg/280px-Zinedine_Zidane_by_Tasnim_03.jpg"
  />
</div>
`;
